src/lazy/encoder/value_writer.rs

```diff
@@ -239,6 +199,20 @@ macro_rules! delegate_value_writer_to_self {
 pub(crate) use delegate_value_writer_to;
 pub(crate) use delegate_value_writer_to_self;
 
+impl<V> ValueWriter for V
+where
+    V: AnnotatableValueWriter,
+{
+    type ListWriter<'a> = <V::ValueWriter as ValueWriter>::ListWriter<'a>;
+    type SExpWriter<'a> = <V::ValueWriter as ValueWriter>::SExpWriter<'a>;
+    type StructWriter<'a> = <V::ValueWriter as ValueWriter>::StructWriter<'a>;
+    type MacroArgsWriter<'a> = <V::ValueWriter as ValueWriter>::MacroArgsWriter<'a>;
+
+    delegate_value_writer_to!(closure |self_: Self| {
+       self_.without_annotations()
+    });
+}
```

Comment:
🗺️ The new blanket implementation of `ValueWriter` for all `AnnotatableValueWriter`s.